### DISCUSSION TITLE
Configurable button

### DISCUSSION THREAD

**Discussion by denizsahin7744** [HomeSpan/HomeSpan#771] [Q&A]
Is there a way to make a button which i can configure in home app?

**Comment by HomeSpan** [comment]
Can you be more specific about exactly what you are trying to configure and how you want it to operate?

**Reply by denizsahin7744** [reply]
I want to make a button which changes the state of all lights in the room

**Reply by HomeSpan** [reply]
Those are called "scenes".  You implement them directly in the Home App - nothing to do with HomeSpan, though you can use a scene to control HomeSpan devices along with commercial devices.  

There's a lot of web sites with instructions on how to create scenes and customize the Home App. A Google search should help you identify all the options.

**Reply by denizsahin7744** [reply]
I know how to use scenes but i want to make a physical button which controls scenes

**Comment by HomeSpan** [comment]
[Example 16 - ProgrammableSwitches](https://github.com/HomeSpan/HomeSpan/blob/master/examples/16-ProgrammableSwitches)

**Reply by denizsahin7744** [reply]
Hi it's again me. Is there a way to make it with ttp223 touch sensor ?

**Reply by HomeSpan** [reply]
I don't have a ttp223 but looked it up on the web here:  https://www.instructables.com/Tutorial-for-TTP223-Touch-Sensor-Module-Capacitive/.  Looks like it's a replacement for a button.  Based on the tutorial, configure your ttp223 to momentary+low and you should be able to use it directly with SpanButton - no modifications needed.  Note you do NOT need to configure this in HomeSpan as a touch sensor.  It's acting like an ordinary pushbutton.
